INSTALL.MD

```diff
@@ -618,6 +618,34 @@ cd  dev_scripts
 ```
 
 f. Then start the traffic generator and MME if applicable.
+
+
+NEW: You can now run ngic without kni support using the following guidelines:
+
+a. Repeat step (a) from previous subsection to setup dpdk driver.
+
+b. Enable -DUSE_AF_PACKET macro in dp/Makefile. Then add the mac addresses of
```

Comment:
Make use of `` to inline format for filenames or setting like `-DUSE_AF_PACKET` and render md better